Yvand.EntraCP/Yvand.EntraClaimsProvider/EntraIDEntityProvider.cs

```diff
@@ -25,16 +26,25 @@ namespace Yvand.EntraClaimsProvider
     public class EntraIDEntityProvider : EntityProviderBase
     {
         public IClaimsProviderSettings Settings { get; }
+        private static List<CachedEntraIDTenantData> CachedTenantData;
+
         public EntraIDEntityProvider(string claimsProviderName, IClaimsProviderSettings Settings) : base(claimsProviderName)
         {
             this.Settings = Settings;
+
+            // Reset cache if config changes
```

Comment:
And when config changes the new instance of `EntraIDEntityProvider` is created, is that so?

Reply:
Exactly. This happens here: https://github.com/Yvand/EntraCP/blob/ddb76a0ca5c3d8d3bd043dad164e0478cc9f6332/Yvand.EntraCP/Yvand.EntraClaimsProvider/EntraCP.cs#L193